metadata.json

```diff
@@ -74,7 +74,7 @@
       "version_requirement": ">= 4.10 < 7.0.0"
     }
   ],
-  "pdk-version": "1.7.1",
+  "pdk-version": "1.8.0",
   "template-url": "file:///opt/puppetlabs/pdk/share/cache/pdk-templates.git",
-  "template-ref": "1.7.1-0-g810b982"
-}
+  "template-ref": "tags/1.8.0-0-g0d9da00"
+}
```

Comment:
please add the missing newline

Reply:
This is managed by PDK, I'd rather not mess with that